### PR TITLE
Add Debian-based copy of the test suite

### PR DESCRIPTION
**Pull request by tianon**
This is useful for testing architectures like mips64le, which aren't supported in Alpine (yet?), as in https://github.com/tianon/gosu/pull/69. :+1: :metal:

### CODE REVIEW

**Comment by tianon** [comment]
```diff
$ diff -u Dockerfile.test-*
--- Dockerfile.test-alpine	2020-03-20 13:58:53.861573498 -0700
+++ Dockerfile.test-debian	2020-03-20 14:06:51.740074771 -0700
@@ -1,7 +1,9 @@
-FROM alpine:3.11
+FROM debian:buster-slim
 
 # add "nobody" to ALL groups (makes testing edge cases more interesting)
-RUN cut -d: -f1 /etc/group | xargs -n1 addgroup nobody
+RUN cut -d: -f1 /etc/group | xargs -n1 -I'{}' usermod -aG '{}' nobody
+# emulate Alpine's "games" user (which is part of the "users" group)
+RUN usermod -aG users games
 
 RUN { \
 		echo '#!/bin/sh'; \
@@ -23,7 +25,7 @@
 COPY gosu /usr/local/bin/
 
 # adjust users so we can make sure the tests are interesting
-RUN chgrp nobody /usr/local/bin/gosu \
+RUN chgrp nogroup /usr/local/bin/gosu \
 	&& chmod +s /usr/local/bin/gosu
 USER nobody
 ENV HOME /omg/really/gosu/nowhere
```
